earth_enterprise/rpms/build.gradle

```diff
@@ -262,13 +263,14 @@ task openGeeCommonRpm(type: GeeRpm, dependsOn: openGeeSharedFiles) {
     autoFindProvides = true
     autoFindRequires = true
 
-    requires('opengee-postgis', '1.5.8', GREATER | EQUAL)
     requires('chkconfig')
```

Comment:
`chkconfig` doesn't have to be added here. It's auto-detected:

```bash
rpm -qpR opengee-common-5.2.0-1.el7.x86_64.rpm | grep chkconfig

chkconfig
```

Reply:
When the `scons release=1 package` target is run as a non-root user, scons does not include  `/sbin` in the path, so auto-detection of `chkconfig` fails.